lib/src/specs/container/box_util.dart

```diff
@@ -286,6 +286,8 @@ final clipBehavior = box.clipBehavior;
 /// - [BoxShadow]
 final elevation = box.elevation;
 
+final radialGradient = box.decoration.gradient.radial;
```

Comment:
In the most part of the best practices and Remix, we are using box.{some attribute} so I think we could follow this pattern.